nodemon.json

```diff
@@ -1,5 +1,7 @@
 {
   "exec": "node ./app/start.js",
   "watch": ["app", "src/govuk"],
+  "ignore": ["**/*.test.*"],
+  "ext": "js",
```

Comment:
Removing json I think will not catch the json file we have here: https://github.com/alphagov/govuk-frontend/blob/main/app/views/full-page-examples/search/data.json

But that's the only example I can find.

Reply:
Ahh good detective work, thank you. Missed that one 👍

Reply:
Added `.json` again 😊